File: 1.data_type/H_Two_numbers.py
import math
a,b = map(int,(input()).split())

f = math.floor(a/b)
c = math.ceil(a/b)
r = round(a/b)
print(f"floor {a} / {b} = {f}")
print(f"ceil {a} / {b} = {c}")
print(f"round {a} / {b} = {r}")

File: 1.data_type/H_Two_numbers.c
#include <stdio.h>
#include <math.h>
int main()
{
    int a, b;
    scanf("%d %d", &a, &b);
    // float div = (float) a/b;

    float floor_ = floor((float)a / b);
    float ceil_ = ceil((float)a / b);
    float round_ = round((float)a / b);

    printf("floor %d / %d = %.0f\n", a, b, floor_);
    printf("ceil %d / %d = %.0f\n", a, b, ceil_);
    printf("round %d / %d = %.0f\n", a, b, round_);

    return 0;
}
